mvlearn/datasets/base.py

```diff
@@ -25,9 +26,17 @@ def load_UCImultifeature(select_labeled="all"):
         specified, all examples in the dataset are returned. Repeated labels
         are ignored.
 
+    shuffle : bool, default=False
+        If ``True``, returns each array with its rows and corresponding
```

Comment:
I think you just need single \` marks for \`True\` to be `True`

Reply:
Same for random_state

Reply:
Single didn't work for me when I tried building the docs for some reason